rust/src/lib.rs

```diff
@@ -2253,17 +2257,22 @@ pub unsafe extern "C" fn zcashlc_decrypt_and_store_transaction(
 
 fn zip317_helper<DbT>(
     change_memo: Option<MemoBytes>,
-    use_zip317_fees: bool,
-) -> GreedyInputSelector<DbT, SingleOutputChangeStrategy> {
-    let fee_rule = if use_zip317_fees {
-        StandardFeeRule::Zip317
-    } else {
-        #[allow(deprecated)]
-        StandardFeeRule::PreZip313
-    };
-    GreedyInputSelector::new(
-        SingleOutputChangeStrategy::new(fee_rule, change_memo, ShieldedProtocol::Orchard),
-        DustOutputPolicy::default(),
+) -> (
+    MultiOutputChangeStrategy<StandardFeeRule, DbT>,
+    GreedyInputSelector<DbT>,
+) {
+    (
+        MultiOutputChangeStrategy::new(
+            StandardFeeRule::Zip317,
+            change_memo,
+            ShieldedProtocol::Orchard,
+            DustOutputPolicy::default(),
+            SplitPolicy::with_min_output_value(
+                NonZeroUsize::new(4).unwrap(),
+                Zatoshis::const_from_u64(10000000),
```

Comment:
Is 1 ZEC potentially a bit high? Current Zcash price is US$37, and AIUI `MultiOutputChangeStrategy` prefers targeting the minimum value over the target number of notes, so for small wallets (coffee shop usage) this is still going to tend towards one note.

Reply:
This is 0.1 ZEC, right?

Reply:
Ah yep, I miscounted. Could we use `1000_0000` to make it clearer?